tests/e2e/test_load.py

```diff
@@ -0,0 +1,60 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import torch
+import ssl
+import functools
+import certifi
+
+from azstoragetorch.io import BlobIO
+
+
+@pytest.fixture(scope="module")
+def ssl_inject_certifi(monkeypatch):
```

Comment:
Are the test currently passing for your environment? I'm not seeing the fixture being referenced anywhere so I'd assume it would not be applied meaning we would be hitting the issues we were seeing in your environment. If they are passing, we should just remove the fixture as that means both of us are fine running the tests without it. We can add it back if the problem comes up again.